samples/subsys/usb/testusb/README.rst

```diff
@@ -0,0 +1,58 @@
+.. _testusb-app:
+
+Testusb application sample
+##########################
+
+The testusb implements a loopback function. This function can be used
+to test USB device drivers and the device stack connected to Linux Host
+and has a similar interface to "Gadget Zero" of the Linux kernel.
+The userspace tool testusb [3] is needed to start the tests.
+
+Building and flashing
+*********************
+
+Follow the general procedure for building and flashing Zephyr device.
+
+Testing
+*******
+
+To run USB Chaper 9 tests:
+
+#. Load the usbtest module on the Linux Host
+
+   .. code-block:: console
+
+      $ sudo modprobe usbtest vendor=0x2fe3 product=0x0100
+
+   The usbtest module should claim the device:
+
+   .. code-block:: console
+
+      [21746.128743] usb 9-1: new full-speed USB device number 16 using uhci_hcd
+      [21746.303051] usb 9-1: New USB device found, idVendor=2fe3, idProduct=0100
+      [21746.303055] usb 9-1: New USB device strings: Mfr=1, Product=2, SerialNumber=3
+      [21746.303058] usb 9-1: Product: Zephyr testusb sample
+      [21746.303060] usb 9-1: Manufacturer: ZEPHYR
+      [21746.303063] usb 9-1: SerialNumber: 0.01
+      [21746.306149] usbtest 9-1:1.0: matched module params, vend=0x2fe3 prod=0x0100
+      [21746.306153] usbtest 9-1:1.0: Generic USB device
+      [21746.306156] usbtest 9-1:1.0: full-speed {control} tests
+
+#. Use the `testusb` tool in `linux/tools/usb` to start the tests
```

Comment:
Usually, when we use a relative path name like this, the assumed root is the Zephyr folder (where Zephyr was cloned).  I don't think this is the case here, so what is the full path?

Reply:
The path is inside Linux kernel source directory